### PR TITLE
docs(changelog): add new changelog about the log-drains compatibility with Logtail

### PR DESCRIPTION
**Pull request by ipfaze**
This PR bring a new changelog dealting about the new log-drains compatibility with Logtail.

### CODE REVIEW

**Comment by ipfaze** [comment]
@EtienneM, I also would like your review about folder and file name of this new kind of changelog. :pray:

**Review by benjaminach — APPROVED** [review]

**Comment by benjaminach** [comment]
Perhaps use "integration" instead of "compatibility"?
Otherwise it is perfect, thanks!

**Review by EtienneM — COMMENTED** [review]
I'm not sure we want a folder inside `changelog` for each feature on the platform. It does not seem really relevant to me. I would add this changelog entry in `features`

**Comment by ipfaze** [comment]
Thanks Étienne, I moved it in the `features` folder.

File: src/changelog/features/_posts/2023-04-05-log-drains-logtail-integration.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 modified_at:	2023-04-05 12:00:00
-title:	'Log-drains are now compatible with Logtail'
+title:	'Log-drains now integrate Logtail'
 ---
 
 ### Changelog
 
-Our log-drains are now compatible with [Logtail](https://betterstack.com/logtail).
+Our "log-drains" now integrate [Logtail](https://betterstack.com/logtail) as a supplier.
```
Comment (benjaminach) [inline src/changelog/features/_posts/2023-04-05-log-drains-logtail-integration.md:8]: "as a third-party provider" ?

**Pull request merged**